.github/workflows/basic.yml

```diff
@@ -128,6 +62,16 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Install oneAPI basekit
+      if: matrix.compiler.cxx == 'icpx'
+      run: |
+        sudo apt update
+        sudo apt install -y gpg-agent wget
+        wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | sudo tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+        echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+        sudo apt update
+        sudo apt install intel-basekit
```

Comment:
actually it's quite a lot of packages being installed within this basekit - shouldn't we just require `intel-oneapi-ippcp-devel` and `intel-oneapi-ipp-devel`...? and perhaps something for `setvars` script (`intel-oneapi-common-vars` or `intel-oneapi-common-oneapi-vars`, I guess)

Reply:
done